src/collapse/perf.rs

```diff
@@ -514,7 +514,9 @@ impl Folder {
                 if !self.cache_line.is_empty() {
                     func.push_str("_[i]"); // inlined
                 } else if self.opt.annotate_kernel
-                    && (module.starts_with('[') || module.ends_with("vmlinux"))
+                    && (module.starts_with('[')
+                        || module.ends_with(".ko")
+                        || module.contains("vmlinux"))
```

Comment:
The updated regex in https://github.com/brendangregg/FlameGraph/pull/232 now allows `vmlinux` anywhere in the string (`vmlinux.*`), so if we want to match the behavior exactly, the `vmlinux` clause would need to be `.contains`.

Reply:
It is

Reply:
You're right. I'm clearly blind.